exec/cnex/bytecode.c

```diff
@@ -109,7 +109,16 @@ void bytecode_loadBytecode(TBytecode *b, const uint8_t *bytecode, unsigned int l
 
     assert(b != NULL);
 
-    if (memcmp(bytecode, "Ne\0n", 4) != 0) {
+    if (memcmp(bytecode, "#!", 2) == 0) {
+        while (i < len && bytecode[i] != '\n') {
+            i++;
+        }
+        if (i+1 >= len) {
```

Comment:
Instead of using `i+1` here and incrementing after, it would be cleaner to increment first, then check `i >= len`.

Reply:
Fixed.